px4_gps_pub_xrce:
  ros__parameters:
    attack_parameters: [
      0.77, 0.05, -1.5, 0.0, 2.0,
    ]
